thehive/info.json

```diff
@@ -0,0 +1,2241 @@
+                "visible": true,
+                "editable": true,
+                "required": false,
+                "onchange": {
+                  "true": [
+                    {
+                      "title": "Zip Password",
+                      "name": "zipPassword",
+                      "type": "password",
+                      "description": "Specify the password for the zip file that you will use to unzip it.",
+                      "tooltip": "Specify the password for the zip file that you will use to unzip it.",
+                      "visible": true,
+                      "editable": true,
+                      "required": true
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        },
+        {
+          "title": "Message",
+          "name": "message",
+          "type": "text",
+          "description": "Specify the message of the observable you want to create for alert in TheHive Platform",
+          "tooltip": "Specify the message of the observable you want to create for alert in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Tags",
+          "name": "tags",
+          "type": "text",
+          "description": "Specify the Tags of the observable you want to create for alert in TheHive Platform",
+          "tooltip": "Specify the source of the observable you want to create for alert in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": true
+        },
+        {
+          "title": "Ignore Similarity",
+          "name": "ignoreSimilarity",
+          "type": "checkbox",
+          "description": "If you select this option, the similarity of the observable type you want to create for the alert in The Hive Platform will be ignored",
+          "tooltip": "If you select this option, the similarity of the observable type you want to create for the alert in The Hive Platform will be ignored",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Additional Field",
+          "name": "additional_field",
+          "type": "json",
+          "description": "Specify the additional fields required for the use case. Provide the fields in JSON format.",
+          "tooltip": "Specify the additional fields required for the use case. Provide the fields in JSON format.",
+          "visible": true,
+          "editable": true,
+          "required": false
+        }
+      ],
+      "enabled": ""
+    },
+    {
+      "title": "Get Observable",
+      "operation": "get_observable",
+      "description": "Retrieves a specific case information from the TheHive platform based on the input parameters you have specified.",
```

Comment:
Please correct the description: Retrieves a specific observable information from the TheHive platform based on the observable ID you have specified.